### ISSUE TITLE
[BUG] VRHUD errors when "Show quick menu hud in desktop" is enabled at launch, breaking the quick menu.

### ISSUE THREAD

**Issue by ShizCalev** [emmVRC/emmVRC-Public#111]
```
[04:44:00.046] [Error] System.NullReferenceException: Object reference not set to an instance of an object

  at emmVRC.Menus.VRHUD+<Loop>d__10.MoveNext () [0x000e7] in <da33ae1905a44ae9909a4e6d0c4bc1ac>:0 

  at MelonLoader.Support.MelonCoroutines.ProcessNextOfCoroutine (System.Collections.IEnumerator enumerator) [0x0000d] in <d052c99cfcf843d3b4a39d240be34bf4>:0 
```

Error only occurs if `Show quick menu hud in desktop` is turned on when the game launches. It results in the quick menu / userlist never updating to show users, position, and network status. Disabling `show quick menu hud in desktop` after the error occurs does not close the quick menu.
![image](https://user-images.githubusercontent.com/6209658/99514437-c2aedd00-2959-11eb-9fdf-16a3aade64e2.png)

If you have the setting disabled when you launch the game, the error doesn't occur, and if you toggle it after, the user list / quick menu hud functions correctly.

(With enabled)
[MelonLoader_20-11-18_04-43-43.676.log](https://github.com/emmVRC/emmVRC-Public/files/5559162/MelonLoader_20-11-18_04-43-43.676.log)

(With disabled)
[MelonLoader_20-11-18_04-44-22.893.log](https://github.com/emmVRC/emmVRC-Public/files/5559159/MelonLoader_20-11-18_04-44-22.893.log)

Similar issue to #88 , however reinstalling redists did not resolve the issue for me.

**Comment by ShizCalev** [comment]
Small update since I haven't been on VR in a while. The above workaround only works for desktop. It's 100% broken when using VR since the VRHUD/quick menu hud always initializes on load (same script error / frozen interface for the quick menu hud)

**Comment by wball22** [comment]
I can second this bug, 100% of the time broken when on VR mode regardless of the slider position, My screen looks the same as @ShizCalev 's does in the picture, side menu not connecting, yet emmvrc states its connected which I assume is confirmable due to the fact that things that require server connections such as emmvrc avatar faves work fine. With no configuration change booting into desktop the feature is fine and i get no errors in the melon loader logs. Attached is a log while in VR.
[MelonLoader_20-12-02_22-56-20.471.log](https://github.com/emmVRC/emmVRC-Public/files/5633978/MelonLoader_20-12-02_22-56-20.471.log)

I was talking to Rakoini on discord about this and we are running the same version of windows 10 20h2, the machines are close to the same with the exception of Rak is using an index and I have a Vive Pro with the wireless adapter if the HMD matters

**Comment by thetrueyoshifan** [comment]
Fixed with emmVRC v2.11.0
